python/send.py

```diff
@@ -417,6 +436,9 @@ def getJobInfo(argv):
             infile = os.path.basename(input_files[i])
             outfile = infile
             print "Name of the input file: ", infile
+            if args.addPileupToSignal :
+                pileupinfile = os.path.basename(pileup_input_files[i])
+                print "Name of the input file for pileup events: ", pileupinfile
```

Comment:
it's not true, further down you pass `all_pileup_inputs` and not just this one file.